leak_report.md

```diff
@@ -1,3 +1,3 @@
 # Leak report
 
-_Use this document to describe whatever memory leaks you find in `clean_whitespace.c` and how you might fix them. You should also probably remove this explanatory text._
+_Functions strip, is_cleaned, and main uses a variable that doesn't relinquish memory. After looking, is_cleaned used a variable from strip that it doesn't let go. So I had to make sure that it doesn't release memory with an empty string, but I altered it so it free's memory after it is done using it.__
```

Comment:
You don't need your answer to be in italics like this. it's not wrong, but it's definitely not necessary. :grin: